src/memex/models/document.py

```diff
@@ -281,15 +281,11 @@ def create_or_update_document_uri(session,
 
     docuri.updated = updated
 
-    if not document.web_uri:
-        parsed = urlparse.urlparse(uri)
-        if parsed.scheme in ['http', 'https']:
-            document.web_uri = uri
-
     try:
         session.flush()
     except sa.exc.IntegrityError:
         raise ConcurrentUpdateError('concurrent document uri updates')
+    return docuri
```

Comment:
Hmm, yeah I don't think so, I just didn't think of `document.document_uris`, I'll give it a try